TCAT/Utilities/Styles.swift

```diff
@@ -46,10 +74,35 @@ extension UIFont {
     
     /// Generate fonts for app usage
     static func style(_ name: String, size: CGFloat) -> UIFont {
+        var name = name
+        if size >= 14 {
+            switch name {
+            case Fonts.SanFrancisco.regular: name = Fonts.SanFrancisco.ProDisplay.regular
+            case Fonts.SanFrancisco.medium: name = Fonts.SanFrancisco.ProDisplay.medium
+            case Fonts.SanFrancisco.semibold: name = Fonts.SanFrancisco.ProDisplay.semibold
+            case Fonts.SanFrancisco.bold: name = Fonts.SanFrancisco.ProDisplay.bold
+            default: break
+            }
+        } else {
+            switch name {
+            case Fonts.SanFrancisco.regular: name = Fonts.SanFrancisco.ProText.regular
+            case Fonts.SanFrancisco.medium: name = Fonts.SanFrancisco.ProText.medium
+            case Fonts.SanFrancisco.semibold: name = Fonts.SanFrancisco.ProText.semibold
+            case Fonts.SanFrancisco.bold: name = Fonts.SanFrancisco.ProText.bold
+            default: break
+            }
+        }
         return UIFont(name: name, size: size)!
     }
     
 }
 
-// MARK: - Spacing
+struct Spacing {
```

Comment:
idt this is necessary. unless our style guide is such that our spacing is consistent within views and across app idt we n2 have values like `eight`, we can just put in `8` for the spacing constraint.

also if we were to do spacing i think it would be better to abstract the `Spacing` in terms of the content and not number. E.g. smtg like this
```
struct Spacing { 
     static let cellTopAndBottomPadding: CGFloat = 8
}
```

But to do that would be intensive b/c we would have to name spacing for every single element and so the list would get quite long. 

Overall think its better to keep spacing as static constants within the view file where the spacing is used

Reply:
That makes sense. But perhaps we could define the constants in this place, and use Spacing.small in each file and give it a better name description. That way if design wanted to double whitespace, we could do so more easily.